src/util.ts

```diff
@@ -53,6 +54,12 @@ export const hasRequiredParameters = (action: ActionInterface): void => {
       "Incorrectly formatted build folder. The deployment folder cannot be prefixed with '/' or './'. Instead reference the folder name directly."
     )
   }
+
+  if (!existsSync(action.folder) && action.folder !== action.root) {
```

Comment:
I'm thinking this may need to be reworked slightly with the recently merged absolute folder paths. What do you think?